src/components/DataDecorator/Decorator/Decorator.js

```diff
@@ -71,29 +74,28 @@ class Decorator extends Component {
       scoreThresholds
     );
     const decoratorClasses = classnames(namespace, classes, className, {
-      [`${namespace}--link`]: this.props.href,
-      [`${namespace}--active`]: this.props.active,
-      [`${namespace}--inert`]: this.props.inert,
-      [`${namespace}--inline`]: this.props.inline,
+      [`${namespace}--link`]: href,
+      [`${namespace}--button`]: onClick,
```

Comment:
Need this class so that I can target it in stylesheet to add a pointer cursor on hover.